packages/wonder-blocks-modal/components/standard-modal.js

```diff
@@ -0,0 +1,113 @@
+// @flow
+import * as React from "react";
+import {StyleSheet, css} from "aphrodite";
+
+import Color from "wonder-blocks-color";
+import {View} from "wonder-blocks-core";
+import {HeadingMedium} from "wonder-blocks-typography";
+
+type Props = {
+    /**
+     * The content of the modal, appearing between the header and footer.
+     */
+    content: React.Node,
+
+    /**
+     * The title of the modal, appearing in the header.
```

Comment:
In the spec the top area is called the titlebar.  The header refers to the colored area in the `with header` variant.  This terminology is different from the current `Modal` component so it's worth calling attention to in the docs.